packages/frontend/src/components/common/LinkButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { AnchorButton } from '@blueprintjs/core';
+import React, { FC } from 'react';
+import { useHistory } from 'react-router-dom';
+
+const LinkButton: FC<
+    { href: string } & React.ComponentProps<typeof AnchorButton>
+> = ({ href, ...rest }) => {
+    const history = useHistory();
+    return (
+        <AnchorButton
```

Comment:
What benefit does this have over the AnchorButton?

Anchor button accepts a href that `.push` or open in new tab on right click. Anything I'm missing?

Reply:
AnchorButton doesn't take advantage of react router, causing a page reload.
By combining the AnchorButton with history.push we have the correct UI, browser let's us open the link in a new tab and update the url path without a page reload.

Reply:
I didn't know using `history.push` prevented a reload!!! That's awesome. TY